src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -67,13 +67,19 @@ export class ServerConnectionIndicator extends LitElement {
       .circle {
         display: inline-block;
 
+        backface-visibility: hidden;
+        -webkit-backface-visibility: hidden;
+
         transition-property: transform, filter, opacity;
         transition-duration: var(--duration);
         transition-timing-function: var(--timing-function);
 
         animation-duration: var(--duration);
         animation-timing-function: var(--timing-function);
         animation-iteration-count: infinite;
+        -webkit-animation-duration: var(--duration);
```

Comment:
I'm not sure if we need these but it's SUCH a pain to get the android emulator in the right state I think we just leave them to be safe